crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1214,6 +1214,18 @@ pub async fn run(
             generate::run(tag, command, &args, child_event)?;
             Ok(0)
         }
+        Command::Info => {
+            CommandEventBuilder::new("info")
+                .with_parent(&root_telemetry)
+                .track_call();
+            let base = CommandBase::new(cli_args.clone(), repo_root, version, color_config);
+
+            match info::run(base).await {
+                Ok(()) => {}
+                Err(_e) => panic!("`info` command failed."),
+            }
```

Comment:
We generally want to avoid `panic` here and if we do panic, we should make sure to include the underlying error.

You can use `?` operator here to bubble up the error if you add a new variant to the cli error type [here](https://github.com/vercel/turborepo/blob/main/crates/turborepo-lib/src/cli/error.rs#L21):
```
#[error(transparent)]
Info(#[from] info::Error),
```
```suggestion
            info::run(base).await?;
```

Reply:
Oh I recommended panic here because the previous message was to open a GitHub Issue and panic includes that advice